docs/UserGuide.md

```diff
@@ -73,21 +73,22 @@ Format: `help`
 
 ### Listing address book records: `list`
 
-Shows a list of all existing properties, sellers, buyers, or clients in the address book.
+Shows a list of all existing sellers, buyers, clients (i.e. buyers and sellers), properties and meetings in the address book.
 
 Format: `list k/KEY`
 
 * The `list` command displays records based on the specified `KEY`. 
-* The `KEY` must be one of the following: `properties`, `buyers`, `sellers`, or `clients`.
+* The `KEY` must be one of the following: `buyers`, `sellers`, `clients`, `meetings` or `properties`.
   * `buyers`: Lists all buyers in the database.
   * `sellers`: Lists all sellers in the database.
   * `clients`: Lists all buyers and sellers (i.e. clients) combined.
   * `properties`: Lists all properties in the database.
+  * `meetings`: Lists all meetings in th e database.
 
 * If an invalid `KEY` is provided, an error message will be displayed.
 
 Key Considerations:
-* Only accepts "properties", "buyers", "sellers", and "clients" (case insensitive) as valid inputs for `k/KEY`.
+* Only accepts "buyers", "sellers", "clients" "properties" and "meetings" (case-insensitive) as valid inputs for `k/KEY`.
```

Comment:
* Only accepts "buyers", "sellers", "clients", "properties" and "meetings" (case-insensitive) as valid inputs for `k/KEY`.


(added a comma between "clients" and "properties")